fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -471,7 +477,7 @@ def reduce(*_: Any) -> None:
                                     group=self.process_group,
                                     async_op=True,
                                 ),
-                                callback=None,
+                                callback=cleanup,
```

Comment:
this should also be in _flush_reduce_calls()

Reply:
I forgot about that, I should store the callback in the bucket actually, that would be  cleaner and make it easier to call it from different places. It's not clear to me that it's what the unit test catches though

Reply:
ok, there was something wrong in the unit test, that second part makes sense